lib/sequel/adapters/drill.rb

```diff
@@ -82,6 +82,15 @@ class Dataset < Sequel::Dataset
       Database::DatasetClass = self
       
       def fetch_rows(sql)
+        # hacks for Sequel functions without adapter methods intended to be extended/overridden
+        
+        # aggregate functions should include backticks
+        sql = sql.sub(/^SELECT count\((.+)?\) AS ([[:graph:]]+)?/, "SELECT count(\\1) AS `\\2`")
```

Comment:
We need to support a more general case here other than defining a few common functions.  Let's change the regex to:

`sql.gsub!(/([[:alnum:]]+\([[:alnum:]]+\)) AS ([[:alnum:]]+)/, '\1 AS `\2`')`